lib/Entrypoint.js

```diff
@@ -2,24 +2,54 @@
 	MIT License http://www.opensource.org/licenses/mit-license.php
 	Author Tobias Koppers @sokra
 */
-"use strict";
+/** @typedef {import("./Chunk")} Chunk */
 
 const ChunkGroup = require("./ChunkGroup");
-
+/**
+ *
+ * @description Entrypoint serves as an encapsulation primitive for chunks that are
+ * apart of a single ChunkGroup. They represent all bundles that need to be loaded for a
+ * single instance of a page. Multipage App Architectures will typically yeild multiple Entrypoint Objects
+ * inside of Compilation, where as a Single Page App, may only contain one with many lazy loaded chunks.
```

Comment:
`apart` -> `part`
`Multipage App Architectures` -> `Multi-page application architectures`
`yeild` -> `yield`
`Entrypoint Objects` -> `Entrypoint objects`
`of Compilation` -> `the compilation`
`where as` -> `whereas`
`Single Page App,` -> remove comma
`lazy loaded` -> `lazy-loaded`

Reply:
Thank you 🙇 I'll tackle these.